doc/run_chaos_mesh.md

```diff
@@ -0,0 +1,79 @@
+# Run Chaos Mesh
+
+Now that you have deployed Chaos Mesh in your environment, in this document, 
+you will learn about how to use it for your chaos experiments.
+
+## Deploy target cluster
+
+After Chaos Mesh is deployed, you can deploy the target cluster to be tested. For illustration purposes,  TiDB is used as a sample cluster.
+
+You can follow the instructions in the following two documents to deploy a TiDB cluster:
+
+* [Deploy using kind](https://pingcap.com/docs/stable/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind/)
+* [Deploy using minikube](https://pingcap.com/docs/stable/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-minikube/)
+
+## Define the experiment config file
+
+The chaos experiment configuration is defined in a ·`.yaml` file. In the following sample file, 
+`pod-kill-example.yaml` defines a chaos experiment to kill one random TiKV pod every 60 seconds:
+
+```yaml
+apiVersion: pingcap.com/v1alpha1
+kind: PodChaos
+metadata:
+  name: pod-failure-example
+  namespace: chaos-testing
+spec:
+  action: pod-failure # the specific chaos action to inject; supported actions: pod-kill/pod-failure
+  mode: one # the mode to run chaos action; supported modes are one/all/fixed/fixed-percent/random-max-percent
+  duration: "60s" # duration for the injected chaos experiment
+  selector: # pods where to inject chaos actions
+    namespaces:
+      - tidb-cluster-demo  # the namespace of the system under test (SUT) you've deployed
+    labelSelectors:
+      "app.kubernetes.io/component": "tikv" # the label of the pod for chaos injection
+  scheduler: # scheduler rules for the running time of the chaos experiments about pods.
+    cron: "@every 5m"
+```
+
+## Create a chaos experiment
```

Comment:
Actually, in this step we are applying/executing an experiment, right?

Reply:
yes, `apply` action means creating and running an experiment.

Reply:
So should we update the heading to maybe "Start a chaos experiment"